ci/environment-3.7.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - pytest-cov
```

Comment:
Since we're dropping Python 3.7 from CI we should be able to safely drop this file too